exonum/src/lib.rs

```diff
@@ -18,6 +18,15 @@
 
 #![deny(missing_debug_implementations, missing_docs, unsafe_code, bare_trait_objects)]
 #![cfg_attr(feature = "long_benchmarks", feature(test))]
+#![cfg_attr(feature = "cargo-clippy", deny(clippy_pedantic))]
+#![cfg_attr(
```

Comment:
Lets add an explanation why are these lints are disabled. For example:
`slutter` and `similar_names` produce too much noise, `cast_possible_wrap`, `cast_possible_truncation` and `cast_sign_loss` don't give alternatives, `use_self` - can be enabled when https://github.com/rust-lang-nursery/rust-clippy/issues/2894 is fixed.